hackathon_site/event/serializers.py

```diff
@@ -107,3 +109,18 @@ class Meta:
             "updated_at",
             "profiles",
         )
+
+
+class UserAcceptanceSerializer(serializers.ModelSerializer):
+    acceptance = serializers.SerializerMethodField()
+
+    class Meta:
+        model = User
+        fields = ("id", "first_name", "last_name", "email", "acceptance")
+
+    @staticmethod
```

Comment:
Yeah good question! You are correct, you can use the function without declaring an instance of UserAcceptanceSerializer. You'd use it this way: 
```
from event.serializers import UserAcceptanceSerializer
UserAcceptanceSerializer.get_acceptance(user=<a user object>)
```
the static method decorator isn't entirely needed here because the function just needs to get the acceptance for the serializer. But since the function doesn't rely on the class, it's a static method anyways.

Reply:
But why we still need `UserAcceptanceSerializer` in front of `get_acceptance(user=<a user object>)` ?

Reply:
because that's what the @staticmethod decorator does for us. if we wanted to make a static method that didn't need the class, i'd put it outside the class but we still need it for the serializer to get the acceptance field.

Reply:
got it!!